apps/omg_child_chain/test/omg_child_chain/block_queue/core_test.exs

```diff
@@ -524,11 +527,31 @@ defmodule OMG.ChildChain.BlockQueue.CoreTest do
 
       # Despite Ethereum height changing multiple times, gas price does not grow since no new blocks are mined
       Enum.reduce(4..100, state, fn eth_height, state ->
-        {_, state} = set_ethereum_status(state, eth_height, 3, false)
+        {_, state} = set_ethereum_status(state, eth_height, 2 * @child_block_interval, false)
         assert expected_max_price == state.gas_price_to_use
         state
       end)
     end
+
+    @tag fixtures: [:empty_with_gas_params]
+    test "Gas price doesn't change if no new blocks are formed, and is lowered the moment there's one",
+         %{empty_with_gas_params: state} do
+      expected_price = state.gas_price_to_use
+
+      # Despite Ethereum height changing multiple times, gas price does not grow since no new blocks are mined
+      Enum.reduce(4..100, state, fn eth_height, state ->
+        {_, state} = set_ethereum_status(state, eth_height, 5 * @child_block_interval, false)
+        assert expected_price == state.gas_price_to_use
+        state
+      end)
+
+      {_, state} =
+        state
+        |> enqueue_block(<<0>>, 6 * @child_block_interval, 1)
```

Comment:
Consider to give friendly name to `5|6 * @child_block_interval`, e.g. 
- `last_mined_plasma_block = 5 * ...`
- `next_plasma_block = 6 * ...`